### ISSUE TITLE
[A11y] Address A11y issues on Data Science Projects

### ISSUE THREAD

**Issue by andrewballantyne** [opendatahub-io/odh-dashboard#965]
This ticket will look to address issues of an accessibility (a11y) nature in the Data Science Projects (DS Projects) section of the main nav.

> Get the [Chrome Dev Tools Axe Extension](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
> Running this extension on the page first loaded and as you use the page will help reveal what needs to be fixed

## Areas covered by this ticket

Main nav: Data Science Projects

> Note this covers all pages within' as well. See data setup for more information how to reveal the sub pages.

Key areas to make sure to test:

* Empty DS Projects page ☑️
* 1-n projects added ☑️
* Project details of one of the pages
         * Empty all sections (aka, new project) ☑️
         * Each section filled with data ☑️
                  * Workbenches, Cluster Storage, Data Connections, Model Serving ☑️
          * Most sections have an expandable row for each row of the table -- more content shows up when you expand it ☑️
        * Each row should have a kebab -- those typically have followup actions ☑️
        * There are several modals; most are associated to the kebab actions or the secondary buttons along the page ☑️
        * Stopping a Workbench has a modal -- once a Workbench is created, there is a toggle switch in the row -- click off

## Data setup for testing

* Creating a project is done from the main nav page -- you should have a button to create a project and it should turn the empty state into a table -- after 10 items, it becomes paginated
* Inside a project
    * Creating a workbench => name can be anything, image selection is needed, typically the first one is fine, everything else should have a default
        * There are expandable items and hidden items, click on the various links and select things in the dropdowns -- you'll likely need to run the axe test on each stage for coverage
    * Create Cluster Storage => It's a modal, just need a name
       * A Workbench can have multiple storages on different paths -- a storage cannot be associated with multiple workbenches
       * A storage can be added to a workbench via the expanded workbench row -- it also requires the storage to be unassociated to any workbench (another modal)
   * Create Data Connection => The data in here is not important, but the name, id, and key will need to be filled out before it can be submitted
       * Data Connections can be connected to multiple workbenches, but workbenches cannot have more than one data connection
   * Models and model servers
       * Configuring a server is pretty straight forward, there are additional fields to expand using the checkboxes
       * Once configured (only one can be at this time), you'll be able to deploy several models
           * Models need a Data Connection & the details of this field is not important for axe testing
